components/inputs/docs/input-text.md

```diff
@@ -77,6 +77,7 @@ The `<d2l-input-text>` element is a simple wrapper around the native `<input typ
 | `aria-invalid` | String | Indicates that the input value is invalid |
 | `autocomplete` | String | Specifies which types of values [can be autofilled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) by the browser |
 | `autofocus` | Boolean | When set, will automatically place focus on the input |
+| `control-instructions` | String | Additional information relating to how to use the control |
```

Comment:
I'm good with the name, but we don't _typically_ refer to these as "controls" anymore, but "components". Would `instructions` or `usage-instructions` work?

Reply:
Yep... agreed. Will tweak.